daemon/algod/api/server/v2/test/handlers_test.go

```diff
@@ -126,6 +129,100 @@ func TestGetBlock(t *testing.T) {
 	getBlockTest(t, 0, "bad format", 400)
 }
 
+func TestGetLedgerStateDelta(t *testing.T) {
+	partitiontest.PartitionTest(t)
+	t.Parallel()
```

Comment:
AFAIK `t.Parallel` doesn't do anything unless you're using `t.Run`. So these can be removed.

Reply:
@winder Can you share how reached the conclusion?

From my understanding, a top-level `t.Parallel()` declaration enables the test be run in parallel with other tests.  Removing the declaration requires a sequential invocation.

I based my understanding on:
* Docs for the method - https://pkg.go.dev/testing@go1.17.13#T.Parallel
* Expected usage in https://github.com/kunwardeep/paralleltest.  We're in the process of enabling this linter more broadly.

Reply:
Thanks, I learned something from this. Please disregard the suggestion.